spinnman/transceiver.py

```diff
@@ -1077,7 +1077,7 @@ def _try_to_find_scamp_and_boot(
         current_tries_to_go = tries_to_go
         while version_info is None and current_tries_to_go > 0:
             try:
-                version_info = self.get_scamp_version()
+                version_info = self.get_scamp_version(n_retries=3)
```

Comment:
Why is this place 3 when nowhere else is?

Reply:
Oh, because it's for boot.

Reply:
I'm beginning to sound like @alan-stokes but perhaps this should be a constant called BOOT_RETRIES?

Reply:
Yes, it probably should...

Reply:
I have added this constant now...